docs/circuit-test.md

```diff
@@ -1,9 +1,11 @@
 # Circuit Test
+
 **Management :material-menu-right: Circuit Test**
 
-A **Circuit Test** can aid troubleshooting by providing details such as False Answer Supervision (FAS), Real-time Transfer Protocol (RTP) quality, Mean Opinion Score (MOS), release reason, jitter, packet loss percentage, and answer delay measurement. The test initiates a call on the switch which is routed out to a termination provider, through the PSTN network. It's then routed to an origination provider which routes the call back to ConnexCS and a specific DID provisioned on the system. This is considered active testing (the call is actively placed on the network) as opposed to passive testing such as [**Stats**](https://docs.connexcs.com/customer/stats/) and [**Reports**](https://docs.connexcs.com/report/). 
+A **Circuit Test** can aid troubleshooting by providing details such as False Answer Supervision (FAS), Real-time Transfer Protocol (RTP) quality, Mean Opinion Score (MOS), release reason, jitter, packet loss percentage, and answer delay measurement. The test initiates a call on the switch, which routes out to a termination provider through the PSTN network. It's then routed to an origination provider, which routes the call back to ConnexCS and a specific DID provision on the system. Its considered active testing (the call is actively placed on the network) as opposed to passive testing such as [**Stats**](https://docs.connexcs.com/customer/stats/) and [**Reports**](https://docs.connexcs.com/report/).
```

Comment:
Please change Real-time Transfer Protocol (RTP) to Real-time Transport Protocol (RTP).
thanks

Reply:
Changed "Transfer" to "Transport"